rules/dead-code/tests/Rector/Return_/RemoveDeadConditionAboveReturnRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Rector\DeadCode\Tests\Rector\Return_\RemoveDeadConditionAboveReturnRector\Fixture;
+
+class Fixture
+{
+    public function run()
+    {
+        if (1 === 1) {
+            return 'yes';
+        }
+
+        return 'yes';
```

Comment:
I see. This case should be covered too, right?

```diff
-if (1 === 'hey') {
-    return 'yes';
-}

 return 'yes';
```

Reply:
the check is not in the cond of if, but on its stmt, so that's already covered.

Reply:
I added new fixture https://github.com/rectorphp/rector/pull/5194/commits/08a197fef44aa782690191ef7414edc1a5ecb6e0 for different cond